cmd/go-aws-sso/main.go

```diff
@@ -83,6 +83,11 @@ func main() {
 					Aliases: []string{"a"},
 					Usage:   "The account id where your role lives in",
 				}),
+				altsrc.NewStringFlag(&cli.StringFlag{
```

Comment:
I would rather like to have the profile setting on an app wide level, not only bound to the assume command. This way one can use `refresh --profile foobar`, too. What do you think?